community/contributing/docs/github-changes.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: single
+title:  "Make a Change Using the GitHub UI"
+sidebar:
+  nav: community
+---
+
+{% include toc %}
+
+# Open a pull request on GitHub
+
+1. On the page where you see the issue, select the pencil icon at the top right. You can also scroll to the bottom of the page and select **Edit this page**.
+1. Make your changes in the GitHub markdown editor.
+1. Below the editor, fill in the **Propose file change** form.
+
+    In the **first field**, give your commit message a title that explains what your pull request is about.
+
+    The Spinnaker repositories use a PR title checker, so the title of your PR must follow a specific format: `<type>(<scope>): <subject>`
+
+    Make sure you include a space after the colon.
+
+    For example:
+
+        docs(plugins): add documentation for plugin creators
+
+    For more information, see [commit message guidelines](https://www.spinnaker.io/community/contributing/submitting/#commit-message-conventions).
+
+    In the **second field**, provide a clear and detailed description of your PR. Please do not to leave these feilds blank, as it is helpful to reviewers when merging your request to have additional context as to what your pull request is about.
```

Comment:
Remove `Please`

Reply:
Done! I've made this change and it's now a new commit.